util/src/main/scala/scala/scalanative/util/ShowBuilder.scala

```diff
@@ -3,12 +3,12 @@ package util
 
 import scala.language.implicitConversions
 
-final class ShowBuilder {
-  private var sb          = new java.lang.StringBuilder
+sealed trait ShowBuilder {
+  def out: Appendable
```

Comment:
`out` should be `protected`, including its implementations in the subclasses. Users of `ShowBuilder` should not have access to it.